### PR TITLE
fix: update testing docs to match reality after #8569

### PR DESCRIPTION
**Pull request by janl**

### CODE REVIEW

**Review by SourceR85 — DISMISSED** [review]
Isn't that easy on non-Debian based distros. (e.g. Fedora, Arch, RHEL, just to name a few other common distros)

Please add a link for others:
https://playwright.dev/docs/docker

Discussion and details can be found here:
https://github.com/microsoft/playwright/issues/26482

**Comment by janl** [comment]
> Isn't that easy on non-Debian based distros. (e.g. Fedora, Arch, RHEL, just to name a few other common distros)

npx works everywhere. I just put into code what actually happens after we moved to playwright from selenium.

If you want to add additional documentation, feel free to push commits to this branch or open a new PR.

File: TESTING.md [hunk 1]
```diff
@@ -232,9 +232,18 @@ Sets the number of iterations each test uses by default.
...
+```shell
+npx playwright install
+```
+
+This will download the `firefox`, `chromium` and `webkit` `CLIENT`s onto
+your system.
+
+PouchDB is tested with `CLIENT=firefox`, `CLIENT=chromium` and `CLIENT=webkit`
+to run a set of tests in the browser automatically. This runs these browsers
+in a “headless” mode and prints the test results back into the terminal.
```
Comment (alxndrsn) [inline TESTING.md:246]: Does this paragraph contain enough info for someone unfamiliar with the project?  Would including the test command in the examples be more helpful?  E.g. `CLIENT=firefox npm test`

Comment (janl) [reply TESTING.md:246]: that’s fair! will fix

File: TESTING.md [hunk 2]
```diff
@@ -232,9 +232,20 @@ Sets the number of iterations each test uses by default.
...
 ### Running tests in the browser
 
-Normally we use `CLIENT=firefox` to run a set of tests in the browser
-automatically. This opens a browser window, automatically runs the requested
-tests in it, and reports the results back to the shell.
+To run tests in the browser, you first have to install playwright:
+
+```shell
+npx playwright install
+```
```
Comment (alxndrsn) [inline TESTING.md:239]: i'm not sure this is the required command - ci is calling `npx playwright install-deps`:

https://github.com/pouchdb/pouchdb/blob/7d71f444e61cc6b4d5da14998a959bb62a0bfb09/.github/actions/install-playwright/action.yml#L7-L9

Comment (alxndrsn) [reply TESTING.md:239]: Also does playwright already prompt for this or return a helpful error message if the dependencies aren't already available?  If that's the case, I wouldn't bother duplicating that documentation here.

Comment (janl) [reply TESTING.md:239]: without this line, I get this error:

```
CLIENT=firefox npm test
…
Error starting tests: browserType.launch: Executable doesn't exist at /Users/jan/Library/Caches/ms-playwright/firefox-1440/firefox/Nightly.app/Contents/MacOS/firefox
╔═════════════════════════════════════════════════════════════════════════╗
║ Looks like Playwright Test or Playwright was just installed or updated. ║
║ Please run the following command to download new browsers:              ║
║                                                                         ║
║     npx playwright install                                              ║
║                                                                         ║
║ <3 Playwright Team                                                      ║
╚═════════════════════════════════════════════════════════════════════════╝
    at startTest (/Users/jan/Work/js/pouchdb/bin/test-browser.js:213:51)
    at checkReady (/Users/jan/Work/js/pouchdb/bin/dev-server.js:109:5)
    at Server.<anonymous> (/Users/jan/Work/js/pouchdb/bin/dev-server.js:103:5) {

```

so I’mma leave it

**Review by SourceR85 — APPROVED** [review]

**Pull request merged**